packages/hdwallet-native/src/crypto/isolation/adapters/bitcoin.ts

```diff
@@ -23,6 +24,10 @@ export class ECPairAdapter implements SignerAsync, ECPairInterfaceAsync {
     this._network = network;
   }
 
+  protected static async prepare() {
+    await networksReady;
+  }
```

Comment:
I don't like this pattern. There's no documentation explaining why `prepare` exists and it's unclear to a developer that `prepare` must be called before they can read from `networks`.

Reply:
I don't either. `create()` is a hack in the first place, I wish you could do e.g. `async constructor(): Promise<this> { await super() }`. If there's a better pattern we should look into it.

In the meantime, I've added extra comments as a stopgap.